camera_calibration/src/camera_calibration/camera_calibrator.py

```diff
@@ -363,8 +363,11 @@ def on_model_change(self, model_select_val):
         self.c.set_cammodel( CAMERA_MODEL.PINHOLE if model_select_val < 0.5 else CAMERA_MODEL.FISHEYE)
 
     def on_scale(self, scalevalue):
-        if self.c.calibrated:
+        if self.c.calibrated and self.c.camera_model is not CAMERA_MODEL.FISHEYE:
             self.c.set_alpha(scalevalue / 100.0)
+        else:
```

Comment:
By addressing my comment, you changed the logic. Beforehand, we were not doing anything if not self.c.calibrated. Now we are warning and setting scalevalue. Is it correct?